test/stat_tracker_test.rb

```diff
@@ -127,6 +127,29 @@ def test_it_can_get_average_goals_per_game_per_team_hoa
     assert_equal expected_avg_away_goals_p_game_p_team, @@stat_tracker.average_goals_per_game_per_team(exclude)
   end
 
+  def test_it_can_find_a_team
+    team3 = @@stat_tracker.teams.find { |team| team.id == 3 }
+
+    assert_equal team3, @@stat_tracker.find_team(3)
+  end
+
```

Comment:
Since the .find_team method also uses `teams.find { |team| team.id == team_id }` isn't this kind of like asserting equal to itself? Maybe instead of creating a variable to assert to, you could `assert_equal 10, @@stat_tracker.find_team(3).franchise_id` or something like that?

Reply:
Yesss!! That's such a simple solution. I didn't feel great about this test so thank you. It's updated now.